### PR TITLE
Feature/97347 create public task entity and store in db

### PR DESCRIPTION
**Pull request by huibvanstraten**
https://ritense.tpondemand.com/entity/97347-public-task-plugin-action-create-public

### CODE REVIEW

File: publictask/backend/plugin/src/main/kotlin/com/ritense/valtimo/backend/plugin/domain/PublicTaskEntity.kt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    @Id
+    @field:Column(name = "public_task_id")
+    val publicTaskId: UUID = UUID.randomUUID(),
+
+    @field:Column(name = "user_task_id")
+    val userTaskId: UUID = UUID.randomUUID(),
+
+    @field:Column(name = "assignee_candidate_contact_data")
+    val assigneeCandidateContactData: String = "",
```
Comment (Niek-Ritense) [inline publictask/backend/plugin/src/main/kotlin/com/ritense/valtimo/backend/plugin/domain/PublicTaskEntity.kt:37]: why do you want a empty string in here?

File: publictask/backend/plugin/src/main/kotlin/com/ritense/valtimo/backend/plugin/repository/PublicTaskRepository.kt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.ritense.valtimo.backend.plugin.repository
+
+import com.ritense.valtimo.backend.plugin.domain.PublicTaskEntity
+import org.springframework.data.jpa.repository.JpaRepository
+import java.util.*
```
Comment (Niek-Ritense) [inline publictask/backend/plugin/src/main/kotlin/com/ritense/valtimo/backend/plugin/repository/PublicTaskRepository.kt:5]: please remove .*

File: publictask/backend/plugin/src/main/resources/config/liquibase/changelog/add-public-task-plugin-entity.xml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            <column name="public_task_id" type="${uuidType}">
+                <constraints  primaryKey="true" nullable="false"/>
+            </column>
+            <column name="user_task_id" type="${uuidType}">
+                <constraints nullable="false"/>
+            </column>
+            <column name="assignee_candidate_contact_data" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="ttl" type="VARCHAR(5)">
```
Comment (Niek-Ritense) [inline publictask/backend/plugin/src/main/resources/config/liquibase/changelog/add-public-task-plugin-entity.xml:33]: Please change the name to time_to_live

File: publictask/backend/plugin/src/main/kotlin/com/ritense/valtimo/backend/plugin/domain/PublicTaskEntity.kt (hunk 1 above)
Comment (huibvanstraten) [reply publictask/backend/plugin/src/main/kotlin/com/ritense/valtimo/backend/plugin/domain/PublicTaskEntity.kt:37]: This is a way Spring gets a default constructor

**Review by Niek-Ritense — APPROVED** [review]

**Pull request merged**